docs/contents/user-docs/install.md

```diff
@@ -112,7 +54,7 @@ helm install --namespace $ACK_K8S_NAMESPACE ack-$SERVICE-controller \
     $CHART_EXPORT_PATH/ack-$SERVICE-controller
```

Comment:
`kubectl create namespace` should be removed. Helm 3 supports a `helm install --create-namespace --namespace ...` option that's preferable.